index.html

```diff
@@ -887,6 +935,11 @@
             techniques</a> are providing.
           </p>
         </li>
+        <li>
+          <p>
+            The <a>solution</a> SHOULD allow developers to specify that no image be downloaded or shown at specified (or implied) breakpoints.
```

Comment:
Again, I would remove "breakpoints". Authors should say which images are not critical. Cancelling load at certain times might actually hurt performance, so this is best left to browsers (because they can turn on/off radio at the right time).